Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/overview/ContactDiaryOverviewFragment.kt

```diff
@@ -0,0 +1,61 @@
+package de.rki.coronawarnapp.contactdiary.ui.overview
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.View
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.contactdiary.ui.ContactDiaryActivity
+import de.rki.coronawarnapp.contactdiary.ui.overview.adapter.ContactDiaryOverviewAdapter
+import de.rki.coronawarnapp.databinding.ContactDiaryOverviewFragmentBinding
+import de.rki.coronawarnapp.util.di.AutoInject
+import de.rki.coronawarnapp.util.ui.observe2
+import de.rki.coronawarnapp.util.ui.viewBindingLazy
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModelFactoryProvider
+import de.rki.coronawarnapp.util.viewmodel.cwaViewModels
+import javax.inject.Inject
+
+class ContactDiaryOverviewFragment : Fragment(R.layout.contact_diary_overview_fragment), AutoInject {
+
+    @Inject lateinit var viewModelFactory: CWAViewModelFactoryProvider.Factory
+    private val vm: ContactDiaryOverviewViewModel by cwaViewModels { viewModelFactory }
+    private val binding: ContactDiaryOverviewFragmentBinding by viewBindingLazy()
+
+    @Inject lateinit var contactDiaryOverviewMenu: ContactDiaryOverviewMenu
```

Comment:
very minor: I prefer to have injects next to each other at the top of the Fragment/Activity (kind of like a constructor)

Reply:
We agreed that we don't put personal preferences to PRs. 
However I changed the position for Axel.
done.